src/layup/orbitfit.py

```diff
@@ -182,16 +193,41 @@ def orbitfit_cli(
 
     chunks = _create_chunks(reader, chunk_size)
 
-    cache_dir = cli_args.ar_data_file_path
+    if cli_args is not None:
+        cache_dir = cli_args.ar_data_file_path
+    else:
```

Comment:
Now on line 67-73 is setting kernels_loc but cache_dir is being change or set here but then how is that even being used if there's kernels_loc?